src/scripts/modules/media/body/content-baked.less

```diff
@@ -122,36 +143,83 @@ h1.example-title .text {
 
 // Helper mixin for expanding all the selectors
 .make-block(@type) {
+  #blockish>.style(@type);
```

Comment:
@philschatz whenever I remove L145-L186 because `.make-block(@type)` is already called in content-common, it does not work. for some reason, the styles in content-baked for notes aren't overriding those in content-common unless I define `.make-block(@type)` again